features/step_definitions/read_entire_post.rb

```diff
@@ -0,0 +1,47 @@
+Given /^I am on the home page$/ do
+  @page = CreationPage.new @browser
+  @page.goto
+  @page.submit_two_posts
+  @home = ListingPage.new @browser
+  @home.goto
+  @reading = ReadingPage.new @browser
+end
+
+When /^I click on post title$/ do
+  @reading.goto
+end
+
+Then /^I should see the entire post$/ do
+  expect(@reading.html).to include("Reading")
+end
+
+Given /^I am on the article page$/ do
+  @page = CreationPage.new @browser
+  @page.goto
+  @page.submit_two_posts
+  @home = ListingPage.new @browser
+  @home.goto
+  @reading = ReadingPage.new @browser
+end
+
+When /^the page loads$/ do
+  @reading.goto
+end
+
+Then /^I should see the author name$/ do
+  expect(@reading.html).to include("Suke Hozumi")
+end
+
+Then /^I should see the title$/ do
+  expect(@reading.html).to include("My Blog Title")
+end
+
+When /^the home page button is pressed$/ do
+  @reading.goto
```

Comment:
This is an unnecessary step. You shouldn't need to navigate to the reading page before clicking the home button. You should just click the home button.